api/flags/flags.go

```diff
@@ -0,0 +1,34 @@
+package flags
+
+import (
+	common "github.com/base-org/blob-archiver/common/flags"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/urfave/cli/v2"
+)
+
+const EnvVarPrefix = "BLOB_API"
+
+var (
+	ListenAddressFlag = &cli.StringFlag{
+		Name:    "api-listen-address",
+		Usage:   "The address to list for new requests on",
+		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "LISTEN_ADDRESS"),
+		Value:   "0.0.0.0:8000",
+	}
+)
+
+func init() {
+	var flags []cli.Flag
```

Comment:
this temp variable doesn't seem to be adding any clarify -- why not just append directly to the global Flags?